src/simple_model.jl

```diff
@@ -155,6 +155,17 @@ function hess(nlp :: SimpleNLPModel, x :: Vector; obj_weight = 1.0, y :: Vector
   return tril(Hx)
 end
 
+function hess_coord(nlp :: SimpleNLPModel, x :: Vector; obj_weight = 1.0, y :: Vector = [])
+  Hx = sparse(hess(nlp, x, obj_weight=obj_weight, y=y))
+  rows = rowvals(Hx)
+  vals = nonzeros(Hx)
+  cols = Int[]
+  for i = 1:nlp.meta.nvar
+    append!(cols, i*ones(Int, length(nzrange(Hx,i))))
+  end
+  return rows, cols, vals
```

Comment:
It's too bad ForwardDiff only does dense Hessians. ReverseDiffSparse seems quite involved.